app-antivirus/fangfrisch/files/conf

```diff
@@ -0,0 +1,22 @@
+# See https://rseichter.github.io/fangfrisch/ for detailed documentation
+
+[DEFAULT]
+db_url = sqlite:////var/lib/fangfrisch/db.sqlite
+local_directory = /var/lib/clamav
+on_update_exec = clamdscan --reload
+
+[malwarepatrol]
+enabled = yes
```

Comment:
If this requires 'receipt', why default to enabled?

Reply:
A receipt code comes with the cost-free registration at Malwarepatrol, and it would be a waste not to use their signatures.